content/docs/examples/advanced-gateways/ingress-certmgr/index.md

```diff
@@ -176,3 +176,56 @@ Hello version: v1, instance: helloworld-5d498979b6-jp2mf
 {{< /text >}}
 
 Note that you have to use the `--insecure` flag as certificates issued by the "staging" ACME-endpoints aren't trusted.
+
+## Moving to production from staging
+
+Now to switch to the production `letsencrypt` issuer.  First we'll reapply the certificate.
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -f -
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: ingress-cert-staging
```

Comment:
We should probably get rid of "-staging" suffix in the name/secretName everywhere in this example if we're going to reuse same resource/secret for both staging and prod. Or use another Cert/Secret pair for prod if it is possible to hotswap them without having to restart ingressgateway.

Reply:
removing